[NFC] Trim redundant `iree-opt` from lit tests.

hanhanW opened the pull request:
The revision deletes the second `iree-opt` pipeline from the testing commands. It is generated by the following commands:

```bash
cd compiler/src/iree/compiler
sed -i -E 's/(iree-opt[^|]*\|)[^|]*\|/\1/' **/*_folding.mlir

cd Dialect/VM
sed -i -E 's/(iree-opt[^|]*\|)[^|]*\|/\1/' **/*.mlir

cd ../../Modules
sed -i -E 's/(iree-opt[^|]*\|)[^|]*\|/\1/' **/*.mlir
```

benvanik reviewed (approved):
This was originally used to test round-tripping (that the output could be read again) but I'm not sure we found it that useful. Fewer invocations will certainly speed up testing, and making all the files consistent is nice! Thanks!

hanhanW committed:
The revision deletes the second `iree-opt` pipeline from the testing commands. It is generated by the following commands:

```bash
cd compiler/src/iree/compiler
sed -i -E 's/(iree-opt[^|]*\|)[^|]*\|/\1/' **/*_folding.mlir

cd Dialect/VM
sed -i -E 's/(iree-opt[^|]*\|)[^|]*\|/\1/' **/*.mlir

cd ../../Modules
sed -i -E 's/(iree-opt[^|]*\|)[^|]*\|/\1/' **/*.mlir
```

Signed-off-by: hanhanW <hanhan0912@gmail.com>

benvanik reviewed (approved):
thank you!
